pkg/common/client/admin_client.go

```diff
@@ -1,21 +1,14 @@
 package client
 
 import (
-	"context"
 	"fmt"
 
 	"github.com/Shopify/sarama"
 )
 
-func MakeAdminClient(ctx context.Context, clientID string, kafkaAuthCfg *KafkaAuthConfig, saramaSettingsYamlString string, brokers []string) (sarama.ClusterAdmin, error) {
-	saramaConf, err := NewConfigBuilder().
-		WithDefaults().
-		WithAuth(kafkaAuthCfg).
-		WithClientId(clientID).
-		FromYaml(saramaSettingsYamlString).
-		Build(ctx)
-	if err != nil {
-		return nil, fmt.Errorf("error creating admin client Sarama config: %w", err)
+func MakeAdminClient(saramaConf *sarama.Config, brokers []string) (sarama.ClusterAdmin, error) {
```

Comment:
Maybe we can get rid of this function and the file?

Reply:
Good idea; will remove